rest/rest-query-resource-group-resources.yml

```diff
@@ -0,0 +1,20 @@
+# Description
```

Comment:
is it necessary to call out rest in file name? user don't  care about how module implemented, but task it achieved

Reply:
yes, true, i can rename it, it's in rest directory anyway